src/rod/builder/primitives.py

```diff
@@ -74,20 +74,12 @@ def __post_init__(self) -> None:
             AssertionError: If the scale is not a 3D vector.
             TypeError: If the mesh_path is not a str or pathlib.Path.
         """
+        mesh_path = pathlib.Path(self.mesh_path)
```

Comment:
Last nit, this attribute could already be a `pathlib.Path` object:

```suggestion
        mesh_path = self.mesh_path if isinstance(self.mesh_path, pathlib.Path) else  pathlib.Path(self.mesh_path)
```

Reply:
Done in [e1a70b0](https://github.com/ami-iit/rod/pull/37/commits/e1a70b0bcfeb4b12b33e16588964cb3d407d22bc).